.github/workflows/lint-workflows.yml

```diff
@@ -0,0 +1,26 @@
+name: Lint GitHub Actions workflows
+
+on:
+  push:
+    paths:
+      - '**.yml'
```

Comment:
Today I stumbled upon one of our internal plugins that is using `.yml` files as configuration files, not for workflows. If we are going to use this workflow only to lint workflows, we can be more specific here on the path, while if we are going to use this also to lint our actions, maybe would it be cleaner to remove this, leaving the choice to the caller? :thinking:

Reply:
I think the caller will define their own value for `on` because when you use workflow, you redefine only the `jobs` part.